### PR TITLE
Aarch64 android fixes

### PR DESCRIPTION
**Pull request by asajeffrey**
Getting android aarch64 to build again.

---

- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors
- [X] These changes do not require tests because build infra

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/21607)

### CODE REVIEW

**Review by ferjm — CHANGES_REQUESTED** [review]
It seems that an update to `ws` is required

```
./Cargo.lock:1: duplicate versions for package `slab`
	The following packages depend on version 0.3.0 from 'crates.io':
		ws
	The following packages depend on version 0.4.1 from 'crates.io':
		mio
```

**Comment by asajeffrey** [comment]
Sigh, updating ws involves updating the test expectations. I love shaving yaks.

**Comment by asajeffrey** [comment]
Argh, and updating ws drags in mio-extras, which causes a different dupe:
```
./Cargo.lock:1: duplicate versions for package `lazycell`
	The following packages depend on version 0.6.0 from 'crates.io':
		mio
	The following packages depend on version 1.0.0 from 'crates.io':
		mio-extras
```

**Comment by asajeffrey** [comment]
I added an entry for slab to servo-tidy.toml. @ferjm, I think this is the best we can do.

**Review by ferjm — APPROVED** [review]
Ok, sounds good to me.

Let's reference #21426 here, so we don't forget about removing the servo-tidy hack.

Thanks!

**Comment by asajeffrey** [comment]
@ferjm Added a comment to #21426.

@bors-servo r=ferjm

File: support/android/fakeld/fake-ld-arm64.sh [hunk 1]
```diff
@@ -11,4 +11,5 @@ set -o pipefail
 source ./support/android/fakeld/fake-ld.sh
 
 export _GCC_PARAMS="${@}"
-call_gcc "arch-arm64" "aarch64-linux-android" "android-21" "arm64-v8a"
+call_gcc "arch-arm64" "aarch64-linux-android" "android-21" "arm64-v8a" \
+         "aarch64-linux-android"
```
Comment (ferjm) [inline support/android/fakeld/fake-ld-arm64.sh]: @bors-servo r- 

Actually, `aarch64-linux-android` is duplicated here. Is this intentional?

Comment (jdm) [reply support/android/fakeld/fake-ld-arm64.sh]: Yes. There are other linker scripts that pass non-identical values to these parameters iirc.

Comment (ferjm) [reply support/android/fakeld/fake-ld-arm64.sh]: @bors-servo r+

**Pull request merged**